src/utils/time-entry-utils/timeEntryUtils.spec.js

```diff
@@ -43,4 +44,34 @@ describe('timeEntryUtils', () => {
     removeTimecardContextEntry(existingTimeEntries, mockSetTimeEntries, 1);
     expect(mockSetTimeEntries).toHaveBeenCalledWith([timeEntry1]);
   });
+
+  describe('isFinishTimeOnNextDay', () => {
+    it('should return true when finish time is before start time', async () => {
+      const startTime = '12:03';
+      const finishTime = '01:01';
```

Comment:
Can we make this more of a boundary test to test the latest time it can possibly be to be considered the next day. E.g. one minute before start time.
Then maybe another simmilar one which is the same time as the start time which would also return true.
```suggestion
      const finishTime = '12:02';
```